jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/jsp/topic/postList.jsp

```diff
@@ -209,14 +209,16 @@
             <c:choose>
               <%--Controls for the first post, they affect topic--%>
               <c:when test="${isFirstPost}">
-                <jtalks:hasPermission targetId="${topic.branch.id}" targetType="BRANCH"
-                                      permission="BranchPermission.DELETE_OWN_POSTS">
-                  <c:set var="isDeleteButtonAvailable"
-						 value='${topic.containsOwnerPostsOnly}'/>
+                  <c:if test='${userId == topic.topicStarter.id}'>
```

Comment:
Please, modify `PreAuthorize` annotation in `TransactionalTopicModificationService` to prevent deletion of topic by other user through dirrect link